recipes/pyreadstat/meta.yaml

```diff
@@ -39,7 +39,7 @@ build:
   # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
   # See https://conda-forge.org/docs/meta.html#building-noarch-packages for more details.
   # noarch: python
-  number: 18
+  number: 19
```

Comment:
You don’t need to keep bumping the build number while this is in staged recipes. You only need to do this once you are making changes in the feedstock.